### ISSUE TITLE
Přenášení poznámky z Aleph

### ISSUE THREAD

**Issue by CIZKOVAL** [NLCR/SeznamDNNT#364]
Dobrý den,
mám požadavek od paní @lichtenbergova, @Cypzirschova, že by měly být do nového seznamu SDNNT přenášeny i veřejné/neveřejné poznámky z Aleph DNT01.

Konkrétně se jedná o pole **IST_p, POZ**.

Kurátor tuto poznámku vyplňuje při vyřazení díla a pokud by byla přenesena do SDNNT (např. do historie), tak by kurátorům pomohlo, kdyby ji viděli při dalším případném navrhování do seznamu apod.

_Příklady poznámek:_

![IST_P](https://user-images.githubusercontent.com/92719629/162401625-9ae20db2-140c-4abe-9125-15539a4826cb.png)

![POZ](https://user-images.githubusercontent.com/92719629/162401645-459cea7c-041a-4d7b-969f-dd710ebb79f3.png)

![POZ1](https://user-images.githubusercontent.com/92719629/162401660-36882bf2-bf94-452e-8113-eda0c0ecd5ae.png)

**Comment by pavel-stastny** [comment]
@Cypzirschova   Jedná se o pole marc_991a ?

**Comment by smallcubes** [comment]
@Cypzirschova poprosil bych Vás o reakci

**Comment by Cypzirschova** [comment]
Tady poprosím o reakci paní @Lichtenbergova, protože nevím, co v tomto poli je. Vím jen toto:

"Pole 985-999 
Tato pole jsou určena pro nadstandardní údaje jednotlivých knihoven a knihovních systémů. Pole 985-999 mohou knihovny využívat dle svého uvážení, jejich evidence je volitelná, doporučuje se u polí potenciálně využitelných pro více knihoven či systémů."

Případně bych musela vidět nějaký příklad, co v tom poli je.

**Comment by Cypzirschova** [comment]
@pavel-stastny @smallcubes Můžete mi prosím poslat příklad, co v tomto poli je?

**Comment by pavel-stastny** [comment]
@Cypzirschova Poslu mailem ukazku celeho oai xml

**Comment by Cypzirschova** [comment]
- 991 - pole POZ
- 992 - pole IST

Obě pole určitě potřebujeme. Více jsem to rozepsala v mailu.
@Lichtenbergova @pavel-stastny

**Comment by pavel-stastny** [comment]
@Cypzirschova @smallcubes Vytvořil jsem třídu, která umí obohatit historii o poznámku z báze DNT-ALL.  Výsledkem je seznam update json souborů.  Poprosil bych @smallcubes pokud byste mi přenesl produkční data na test (nebo dev),  spustil bych generování update dávek i se skriptem.  Po kontrole by se mohly skripty aplikovat přímo na prudkci. 

Poznámky:  

- Přenášel jsem pouze poznámky, stavy a uživatelé v historii byly
- Poznámka je normálně v historii veřejného a kurátorského stavu, viz: 
<img width="510" height="210" alt="Image" src="https://github.com/user-attachments/assets/cfd5f056-c567-4054-9838-0203c97d5054" />
 
- Obohacuji pouze následnické záznamy, tedy ne ty původní se stavem D

**Comment by Cypzirschova** [comment]
@pavel-stastny Bylo by možné dostat seznam záznamů, kterých se to týká?

**Comment by pavel-stastny** [comment]
@Cypzirschova Ano, pošlu csv soubor. Dle toho co jsem probíral, tak je to necelých 6000 záznamů.

**Comment by Cypzirschova** [comment]
@pavel-stastny Děkuji :-)
